kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriberServiceImpl.java

```diff
@@ -475,6 +475,8 @@ public ChildImportRejection updateChildSubscriber(Long msisdn, MctsChild childUp
                         subscriberByMctsId.setDateOfBirth(dob);
                         if (subscriberByMctsId.getMother() == null) {
                             subscriberByMctsId.setMother(childUpdate.getMother());
+                        } else  if (subscriberByMctsId.getMother() != null && childUpdate.getMother() == null) {
+                            childUpdate.setMother(subscriberByMctsId.getMother());
```

Comment:
Looks like child need not be updated if mother is coming as null -- since childupdate.getmother came as null - we are updating again with old db value. Just check if childUpdate.setMother --doesn't set at all to null -- before calling this method